.php_cs

```diff
@@ -0,0 +1,3 @@
+<?php
+// TO BE REMOVED.
```

Comment:
i think this file must not be in the repository. however, it should be in .gitignore

Reply:
ah, sorry, yes